### DISCUSSION TITLE
How to use KeyDown event in Maui?

### DISCUSSION THREAD

**Discussion by mateusfelixmeta** [dotnet/maui#21557] [Q&A] [answered]
How to use KeyDown event in Maui? I have a button and I want it to be clicked when I press enter.

**Comment by davidortinau** [comment, marked as answer]
I wrote a plugin that provides key events. You might use that approach. Use the plugin or just snag the code. I assume you would execute the same command or event you have bound to that button, not actually try to simulate a click.

Sample: https://github.com/davidortinau/GuessWord
Plugin: https://github.com/davidortinau/Plugin.Maui.KeyListener

Related, if you want to execute a command when Return is pressed on an `Entry` field, there's a prop `ReturnCommand` for that.

```xaml
<Entry Placeholder="Type message"
                        Text="{Binding UserInput}"
                        ReturnCommand="{Binding SendMessageCommand}"
                    />
```

**Reply by inescastro05** [reply]
me baje el ejemplo pero no hace nada, ni en windows ni en android, en android se va siempre al reset sin importar que tecla toque
